TOOLS/mpv-osd-symbols.sfdir/README.md

```diff
@@ -0,0 +1,16 @@
+Steps to add new icons:
+
+- Install `fontforge`
+- Install the last freely licensed version of symbola (`ttf-symbola-free` in the
+  AUR)
+- `fontforge /usr/share/fonts/TTF/Symbola.ttf TOOLS/mpv-osd-symbols.sfdir`
+- Check the Unicode hex value of the desired character (`g-a` in vim)
+- Scroll until that value in the Symbola window and click it
+- Press Ctrl+c
```

Comment:
Describe the action, for example. Not only "press".

`- Copy selected glyph (Ctrl+C)`